### PR TITLE
Add submodule

### PR DESCRIPTION
**Pull request by dwalintukan**
**This PR removes all non-Electron logic out of the repo. `bodhi-graphql` will solely be used for any Electron-related code. All backend-related logic should go in `bodhi-server` now, which can be used as a standalone API, webserver, etc without an GUI.**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "server"]
```
[the remark this answers is not in the document]
Comment (alex-cory) [reply .gitmodules]: Only a suggestion! :)

File: main.js [hunk 2]
```diff
@@ -309,64 +351,90 @@ app.on('window-all-closed', () => {
...
 
-// Load UI when services are running
-server.emitter.once(ipcEvent.SERVICES_RUNNING, () => {
-  if (uiWin) {
-    uiWin.maximize();
-    uiWin.loadURL(`http://${Config.HOSTNAME}:${Config.PORT}`);
-  }
+/* Emitter Events */
+// Show error dialog for any errors from startServer()
+Emitter.emitter.on(ipcEvent.SERVER_START_ERROR, (errMessage) => {
```
[the remark this answers is not in the document]
Comment (dwalintukan) [reply main.js]: Emitter is a class that includes the instance of `emitter`, but also some helper functions as well. Guess I can name `Emitter` to `EmitterHelper` to make it more clear.

File: main.js [hunk 3]
```diff
@@ -128,10 +162,10 @@ function showSelectEnvDialog() {
     message: i18n.get('selectQtumEnvironment'),
     defaultId: 2,
     cancelId: 2,
-  }, async (response) => {
+  }, (response) => {
     switch (response) {
-      case 0: {
-        if (testnetOnly) { // Testnet only
+      case 0: { // Mainnet selected
```
[the remark this answers is not in the document]
Comment (dwalintukan) [reply main.js]: done

File: .gitmodules (hunk 1 above)
[the remark this answers is not in the document]
Comment (dwalintukan) [reply .gitmodules]: Let me look into this. If we do change we'll change in a diff PR

**Comment by dwalintukan** [comment]
going to merge this for today's release

**Pull request merged**